playbooks/edx-east/edxapp.yml

```diff
@@ -9,7 +9,6 @@
   roles:
     - aws
     - role: scorm
-      when: "{{ EDXAPP_XBLOCK_SETTINGS }}.get('ScormXBlock', False)"
```

Comment:
Why are we removing this check? I assumed we only wanted to load scorm when declared in server-vars `EDXAPP_XBLOCK_SETTINGS`